packages/webapp/src/pages/index.tsx

```diff
@@ -82,38 +90,44 @@ export const Index = () => {
                     </Card>
                 </div>
                 <aside className="col-span-1 space-y-4 pb-5">
-                    <CommunityStatsCard memberStats={memberStats} />
+                    {treasuryBalance && (
+                        <CommunityStatsCard
+                            memberStats={memberStats}
+                            treasuryBalance={treasuryBalance}
+                        />
+                    )}
                 </aside>
             </div>
         </RawLayout>
     );
 };
 
-interface CommunityStatusProps {
+interface CommunityStatsProps {
     memberStats: any;
+    treasuryBalance: Asset;
 }
 
-const CommunityStatsCard = ({ memberStats }: CommunityStatusProps) => {
-    return (
-        memberStats && (
-            <Card className="flex flex-col justify-center items-center h-full space-y-8 text-md lg:text-xl">
-                <Heading size={2} className="mb-2">
-                    Community Stats
-                </Heading>
-                <Link href="/members" className="font-medium">
-                    {memberStats.active_members} active member
-                    {memberStats.active_members !== 1 && "s"}
-                </Link>
-                <Link
-                    href="/induction/pending-invitations"
-                    className="font-medium"
-                >
-                    {memberStats.pending_members} pending invitation
-                    {memberStats.pending_members !== 1 && "s"}
-                </Link>
-            </Card>
-        )
+const CommunityStatsCard = ({
+    memberStats,
+    treasuryBalance,
+}: CommunityStatsProps) =>
+    memberStats && (
+        <Card className="flex flex-col justify-center items-center h-full space-y-4 text-md lg:text-lg">
+            <Heading size={2} className="mb-2">
+                Community Stats
+            </Heading>
+            <Text className="font-medium" size="md">
+                {`Treasury: ${assetToString(treasuryBalance, 4)}`}
```

Comment:
I think this could be simplified to: `Treasury: {assetToString(treasuryBalance, 4)}`